services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/client/S3ClientWrapperTest.java

```diff
@@ -241,17 +244,15 @@ void testPutObjectForAmzAclHeader() {
   void putObjectsThrowsObjectStoreOperationFailedExceptionIfClientThrows(Class<Exception> cause) {
     when(s3Client.putObject(any(PutObjectRequest.class), any(RequestBody.class))).thenThrow(cause);
     assertThatExceptionOfType(ObjectStoreOperationFailedException.class)
-        .isThrownBy(
-            () -> s3ClientWrapper.putObject(VALID_BUCKET_NAME, VALID_PREFIX, VALID_PATH, Collections.emptyMap()));
+        .isThrownBy(() -> s3ClientWrapper.putObject(VALID_BUCKET_NAME, VALID_PREFIX, VALID_PATH, emptyMap));
```

Comment:
what was the sonar issue on this?

Reply:
There were more than one single call insise the Assertion: 1.putObject(...) and 2.emptyMap(), so that's why I've extracted the emptyMap() into a variable